### ISSUE TITLE
unhandled promise rejection: undefined

### ISSUE THREAD

**Issue by janwilke** [ioBroker/ioBroker.zigbee#791]
Hello all,

more often I get an error - after automatic restart zigbee is working fine for some hours.

zigbee.1 | 2020-09-25 22:00:49.404 | error | (4369) unhandled promise rejection: undefined
-- | -- | -- | --
zigbee.1 | 2020-09-25 22:00:49.403 | error | Unhandled  promise rejection. This error originated either by throwing inside of  an async function without a catch block, or by rejecting a promise which  was not handled with .catch().
 
Zigbee Adapter 1.2.1.

Conbee II (Updated to last firmware today deCONZ_ConBeeII_0x26650700.bin.GCF)
![zigbee_error](https://user-images.githubusercontent.com/43091772/94313598-49a98f80-ff7f-11ea-9202-7ec8321e2005.png)

Any ideas? THANKS!

**Comment by asgothian** [comment]
Please activate debug mode for the Adapter as well as zigbee Shepherd debug Info and try to recreate the error.

**Comment by martinschm** [comment]
Hi, I have a similar error messages in relation to pairing Heiman smoke detectors (see issue #802 )


2020-10-02 16:05:09.934 - error: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
--
2020-10-02 16:05:09.935 - error: zigbee.0 (1356) unhandled promise rejection: Command 0x00158d0004fd0fd7/1 ssIasZone.enrollRsp({"enrollrspcode":0,"zoneid":23}, {"timeout":10000,"disableResponse":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'No network route' (205))
2020-10-02 16:05:09.935 - error: zigbee.0 (1356) Error: Command 0x00158d0004fd0fd7/1 ssIasZone.enrollRsp({"enrollrspcode":0,"zoneid":23}, {"timeout":10000,"disableResponse":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'No network route' (205))

**Comment by asgothian** [comment]
This error hints at issues with reaching the smoke detector. The zigbee mesh seems to not know where to send the message to. (No network route). Can you repeat the pairing process with the smoke detector close to the zigbee coordinator ?

**Comment by martinschm** [comment]
Hi,

I did not see the message regarding debug until now only your last message.
I re-added the devices and 5 out of 6 are working ok now. Moving them very close to the coordinator worked great.

Now there is only one device causing trouble. I added it already twice with no success yet. Will keep trying.

`
zigbee.0 | 2020-10-05 15:14:40.123 | error | (2185950) DeviceConfigure failed 0x00158d0004b487e0 SmokeSensor-N-3.0, attempt 1 (Error: Bind 0x00158d0004b487e0/1 genPowerCfg from '0x00124b001caab658/1' failed (AREQ - ZDO - bindRsp after 10000ms)
-- | -- | -- | --
zigbee.0 | 2020-10-05 15:14:30.102 | info | (2185950) Configuring 0x00158d0004b487e0 SmokeSensor-N-3.0
zigbee.0 | 2020-10-05 15:14:30.101 | error | (2185950) DeviceConfigure failed 0x00158d0004fcf946 SmokeSensor-N-3.0, attempt 1 (Error: Bind 0x00158d0004fcf946/1 genPowerCfg from '0x00124b001caab658/1' failed (AREQ - ZDO - bindRsp after 10000ms)
unifi.0 | 2020-10-05 15:14:23.737 | info | (11787) Update done
zigbee.0 | 2020-10-05 15:14:20.063 | info | (2185950) Configuring 0x00158d0004fcf946 SmokeSensor-N-3.0
zigbee.0 | 2020-10-05 15:14:20.061 | error | at processTimers (internal/timers.js:492:7))
zigbee.0 | 2020-10-05 15:14:20.061 | error | at listOnTimeout (internal/timers.js:549:17)
zigbee.0 | 2020-10-05 15:14:20.061 | error | at Timeout._onTimeout (/opt/iobroker/node_modules/zigbee-herdsman/dist/utils/waitress.js:46:35)
zigbee.0 | 2020-10-05 15:14:20.061 | error | (2185950) DeviceConfigure failed 0x00158d0004fcf82a SmokeSensor-N-3.0, attempt 1 (Error: Bind 0x00158d0004fcf82a/1 genPowerCfg from '0x00124b001caab658/1' failed (AREQ - ZDO - bindRsp after 10000ms)
zigbee.0 | 2020-10-05 15:14:10.037 | info | (2185950) Zigbee started
`

As a suggestion for the adapter. Would it be possible to catch the error and give a user friendly message in regards to not reaching the device.
